### ISSUE TITLE
The actual display is not as it is in the readme

### ISSUE THREAD

**Issue by badicsalex** [CAD97/colored-diff#8]
The output looks like this:
![image](https://user-images.githubusercontent.com/9196531/181098158-1fd9faa2-6259-4df7-9dd2-3c8b7b747654.png)

Instead of 
![image](https://user-images.githubusercontent.com/9196531/181098221-bed24619-d3cb-4cf1-988a-8cce1b9de1b4.png)
(Screenshot with v0.2.2)
Which is _much_ less readable.

Probably introduced in a690b657091.

**Comment by schwa** [comment]
Yup. It's not really usable with current output. Small changes add a lot of extra lines to output.
